### ISSUE TITLE
What crimes do Mexican citizens commit abroad?

### ISSUE THREAD

**Issue by dbaptistr** [jsoma/data-studio-projects#186]
# Pitch
President Donald Trump says that Mexico only sends its worst people to the United States. Meanwhile, the Mexican government has increased the amount of money to protect Mexicans abroad charged with crimes. What crimes are Mexicans actually involved in the United States and how has that changed between 2014 and 2017?

## Summary
I found a database by the Mexican Ministry of Foreign Affairs which shows the amount of consular assistance it has given to Mexican citizens charged with a crime in the United States. It's possible to see what crimes have been the most common in the last seven years, and might show increases or decreases of them.

## Details

**Possible headline(s):** What crimes do Mexican citizens commit abroad?

**Data set(s):** https://datos.gob.mx/busca/dataset/penal--casos-de-proteccion-y-o-asistencia-consular-atendidos-por-la-rdcm-en-el-mundo

**Code repository:** https://github.com/dbaptistr/data-studio/tree/master/02-project
	
**Possible problems/fears/questions:** I'm trying to determine which kind of graph would work best to show the most common crimes Mexican citizens are charged of and to determine if there's any clear increase or decrease of cases.

## Work so far

I cleaned up the data to show only crimes within the United States and summed up the crimes by year. I have very ugly bar graphs for each crime from 2014 to 2017. I've yet to choose a better way to plot the information and apply my style to the graphs.

![captura de pantalla 2018-07-17 a la s 09 22 04](https://user-images.githubusercontent.com/39470114/42820350-f1685b60-89a3-11e8-8b84-a31956b7fedb.png)
![captura de pantalla 2018-07-17 a la s 09 22 15](https://user-images.githubusercontent.com/39470114/42820353-f2f0bfa4-89a3-11e8-8fcc-c5d3ec214768.png)
![captura de pantalla 2018-07-17 a la s 09 22 23](https://user-images.githubusercontent.com/39470114/42820357-f469b570-89a3-11e8-810c-59ae319639d0.png)
![captura de pantalla 2018-07-17 a la s 09 22 29](https://user-images.githubusercontent.com/39470114/42820361-f6b09146-89a3-11e8-952c-4afd5056a5f1.png)

## Checklist

This checklist must be completed before you submit your draft.

- [ ] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [ ] My issue links to my data set(s)
- [ ] My issue links to my code repository
- [ ] My issue explains what I'd like to explore in the data set
- [ ] My issue includes images - either inspiration or what I've done so far

**Comment by jsoma** [comment]
* Yes, those graphs are very ugly, but the dataset is so interesting that I'm totally sold!
* You'll want to change your font so that àćçêñt marks will display right. For my machine I use "Arial Unicode MS," I forget which notebook that code is in - if you can't find it or no one in Slack knows I can track it down for you.
* For your "compared to what?" you have two options you could think about - Mexicans vs. baseline rates in the US, or Mexicans in the US vs. Mexicans elsewhere

**Comment by dbaptistr** [comment]
# Update
	
## Your project content: images/words/etc
Mexicans in the U.S mainly commit three crimes: those related to illegal immigration, drugs and driving under the influence. I have charts showing their distribution over the past six years. 	

![migration](https://user-images.githubusercontent.com/39470114/42979462-4b92f624-8ba0-11e8-8e90-1b7dc7ac7271.png)
![drug_related](https://user-images.githubusercontent.com/39470114/42979400-f06b0a8e-8b9f-11e8-8e34-a632ffb416b9.png)
![delitos_vehiculos](https://user-images.githubusercontent.com/39470114/42979418-0e41b486-8ba0-11e8-98d6-99da8364048e.png)


  
## Any changes in direction or topic?
  
No.
	
## Problems/Questions
I can't plot the distribution over the years of the crimes. I know the origin of the problem might be on the dataframe, but I can't figure out how to solve it. I haven't made the comparison with crimes in other countries because I can't plot it correctly.

![captura de pantalla 2018-07-19 a la s 22 02 12](https://user-images.githubusercontent.com/39470114/42979373-c875f94e-8b9f-11e8-9edf-185947c543b2.png)
 
![captura de pantalla 2018-07-19 a la s 22 01 58](https://user-images.githubusercontent.com/39470114/42979344-a30a6ffa-8b9f-11e8-8a9a-7135793c22ec.png)

	
## Checklist
	
- [ y] I have included my visuals
- [ y] I have filled out the sections above
- [ y] I have been updating my DIARY.md with details about my process
- [y ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by playfairbot** [comment]
Hi! I'm a little robot, checking in on your project.

You need some feedback, let me summon @linleysanders, @nickospi, @mattrehbein for you

**Comment by mattrehbein** [comment]
I really like the breakdown into the three groups of most-perpetrated crimes. I think it'd help orient me as a reader if the first graph somehow put these three categories together so I could see which is number 1, 2 & 3 and by how much (a graph would really nicely show if illegal immigration is by far the commonly committed crime).

For the graphs themselves, I'd suggest putting the years under the bars; having them in a legend makes my eyes have to jump back and forth too much. I'd also try to kill the unnecessary white space on either side of the bars in the graphics.

Possible breakdown angle to explore might be homicides. That's a topic that would be good to compare against total number of murders or some other measure, as Soma suggested. Seeing how often Mexican citizens are charged in homicide cases in US would fit into your timeframe of the Trump years, and I imagine will act as a fact check for those who focused at length on the [Kate Steinle](http://www.politifact.com/truth-o-meter/article/2017/dec/18/Donald-Trumps-misleading-claims-in-Kate-Steinle/) case.

**Comment by nickospi** [comment]
- I think these would show better as a trend line per crime with annotations for each year on the line and not on a legend.
- like the color scheme and the style 
- I'd prefer the title/subtitle on the left
- There could be a stacked graph with the let's say top 5 crimes all together per year and how this has changed after Trump or not

**Comment by linleysanders** [comment]
This is such an interesting dataset—I really like how you chose a topic that is so relevant and also points out flaws in American media perception of Mexican immigrants. 

- I agree with Max that it would be nice to have the x-axis display the years.
- I think you should consider grouping the charts by crimes committed per year so it would become even more obvious that illegal immigration is far-in-a-way more prevalent than drug or alcohol-related crimes.
- For the headlines, I'd make them more descriptive—especially the one that just says DUI

**Comment by dbaptistr** [comment]
# Update
	
## Your project content: images/words/etc

I made a new chart showing the total number of assistances by crime, which shows illegal immigration and drug related crimes are most common ones committed in the US.

![total_crimes](https://user-images.githubusercontent.com/39470114/43156453-28c20120-8f48-11e8-83a1-c4af9f043e8a.png)

A look into the three most common crimes:

![immigration](https://user-images.githubusercontent.com/39470114/43160710-f62fd17c-8f53-11e8-8d15-2e019a641c02.png)

![drugcrimes](https://user-images.githubusercontent.com/39470114/43165513-111756a0-8f62-11e8-9e33-dc26f9ed73ff.png)

![duicrimes](https://user-images.githubusercontent.com/39470114/43166324-3c187788-8f64-11e8-9706-7e83d3dffb4a.png)

What about murders?
Murder is number ten in the list of crimes committed by Mexicans in the US.

![murders](https://user-images.githubusercontent.com/39470114/43167541-0cbcee7a-8f68-11e8-8731-3c434f590770.png)

What's the difference with crimes committed by Mexicans in other places?

![abroad_total_crimes](https://user-images.githubusercontent.com/39470114/43170459-34047dc6-8f73-11e8-9229-99199fe91d9d.png)
  
## Any changes in direction or topic?
  
No, just following the feedback given to me.
	
## Problems/Questions
  
I finally realized I was having trouble with the indexes in my data frames and that's why I couldn't plot correctly. 
I wonder if the current graphs I have are enough to draw conclusions and if they are informative.
	
## Checklist
	
- [y ] I have included my visuals
- [y ] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [y ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
for your bar charts, there's no need for different colors each year. color should have a meaning and not be decoration. if you want to use it make all the bars one color and then just change the one for this year that you are currently annotating. 

also, if you have a scale on a chart you don't also need numbers. it's like wearing a belt and suspenders.

as to the line chart, we cannot read 13 different categories, just too much to look through and even see as they are stacked on top of each other.

one way to show these data might be to use slope charts...http://datavizproject.com/data-type/slope-chart/

you could break this chart into two pieces perhaps and show us the top 5 on one chart and the bottom five on another.

the only thing that worries me here is there is no 'compared to what' in general. how does this compare to black people or white? i know that is an entirely different data set and different question. but if the numbers are much lower...

**Comment by dbaptistr** [comment]
# Final

## Project visuals/text

![total_crimes_slope](https://user-images.githubusercontent.com/39470114/43352126-c43060f6-91ec-11e8-990a-4ba60d5a602c.png)

![abroad_total_crimes](https://user-images.githubusercontent.com/39470114/43352128-d4494106-91ec-11e8-9491-7cd851ff79e1.png)

![immigration](https://user-images.githubusercontent.com/39470114/43352131-d9d65eb0-91ec-11e8-9a3d-de3c2f4dfe99.png)

![drugcrimes](https://user-images.githubusercontent.com/39470114/43352132-df064df0-91ec-11e8-9fff-a08772e2f04d.png)

![duicrimes](https://user-images.githubusercontent.com/39470114/43352134-e33e1dc6-91ec-11e8-87ff-1577beac00b7.png)


## Details

**Headline:** Immigration is Mexican's biggest crime

**Published website version:** https://dbaptistr.github.io/Crimes_abroad/

**Code repository:**https://github.com/dbaptistr/data-studio/tree/master/02-project

**Final data set(s):** 

## What did you find to be the most difficult part of this project? 
I had very basic issues building my data frames, but I could solve them. I couldn't do the slope chart in Python, but I learnt how to do it in Illustrator.

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I'm not sure if the comparison between crimes in the US and in the rest of the world is enough. I think there's still so much information to actually prove that we don't send our worst people to the US.


## Checklist

- [y ] I have included my visuals
- [y ] I have posted my project to my project website
- [n ] I have been updating my DIARY.md with details about my process
- [y ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by ella24** [comment]
Diana, it is great that you followed Sarah's feedback and used just the top five crimes for the line graph.
For the second graph, I think the reader needs to see what other countries you are talking about. Or a least the geographical region. Maybe do it by ‘Asia’, ‘Europe’, and ‘Latin America’.
In the third one, it would be great to use annotation to explain why it has decreased in 2017 and present…what is happening in the US politics?
Drug relating crimes are on the rise in the USA…there is a story there.
Driving under the influence, also…this is another whole story by itself.
Great job. I am hungry for more!

**Comment by hakantan** [comment]
I like the look of the final product. 

Especially the first graph is clear in its message. A huge chunk of crimes we're talking about are actually related to illegal immigration (and not sex crimes, assault etc.). This is very helpful when having a conversation/national debate on this issue. 

Some minor suggestions:

- With regards to the consular assistance: I think it would not include this chart in the final product. Being in a foreign country and needing assistance can have a variety of reasons, one of them being a tourist and not understanding local law. Which is different from the main idea you seem to be interested in: people actually _living_ in a country, maybe staying there for good. 

- Why did you, in your bar charts, highlight the 2017 bar? That wasn't clear to me.

**Comment by jlstro** [comment]
Hey, so you are done now, maybe my comments are not very useful anymore, anyhow: The colors you chose are nice, I like them. The topic is very interesting and your would be helpful info to some in this crazy, hate-driven debate that's going on in the US. From a reader's perspective, I'd like to know more about the spike in 2013-14, maybe an annotation can help there.
